.github/workflows/ci.yml

```diff
@@ -273,6 +259,14 @@ jobs:
             cargo check --manifest-path "$toml";
           done
 
+  semver:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: wget -q -O- https://github.com/obi1kenobi/cargo-semver-checks/releases/download/v0.22.1/cargo-semver-checks-x86_64-unknown-linux-gnu.tar.gz | tar -xz -C ~/.cargo/bin
```

Comment:
Why is this needed? Doesn't `obi1kenobi/cargo-semver-checks-action@v2` bring `cargo-semver-checks` along with it? Does this speed up CI?

Reply:
See https://github.com/obi1kenobi/cargo-semver-checks-action/issues/47.